programs/spammers.go

```diff
@@ -11,89 +10,144 @@ import (
 	"github.com/iotaledger/evil-tools/spammer"
 )
 
+const (
+	maxBigWalletsCreatedAtOnce = 10
+	checkFundsLeftInterval     = time.Second * 20
+)
+
 var log = logger.New("customSpam")
 
-func CustomSpam(params *CustomSpamParams, accWallet *accountwallet.AccountWallet) {
-	w := evilwallet.NewEvilWallet(evilwallet.WithClients(params.ClientURLs...), evilwallet.WithAccountsWallet(accWallet))
-	wg := sync.WaitGroup{}
+func requestFaucetFunds(params *CustomSpamParams, w *evilwallet.EvilWallet) <-chan bool {
+	if params.SpamType == spammer.TypeBlock {
+		return nil
+	}
+	var numOfBigWallets = 2
+	if params.Duration >= 0 {
```

Comment:
```suggestion
	if params.Duration > 0 {
```

This check can even be removed, since `BigWalletsNeeded` return 1 with duration `0`